src/pages/subreddit/SubredditPage.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import axios from 'axios';
+import Header from '../../components/Header';
+import './SubredditPage.css';
+
+function SubredditPage() {
+    const [details, setDetails] = useState({});
```

Comment:
Als `details` uiteindelijk een object wordt, dan zou ik de default waarde aanpassen naar `undefined`. Dan heb je een duidelijker verschil tussen of het al wél of niet is ingeladen. Dan wordt je check hier beneden ook wat simpeler
```suggestion
    const [details, setDetails] = useState();
```